device-server/src/main/kotlin/com/badoo/automation/deviceserver/DeviceServer.kt

```diff
@@ -233,6 +233,16 @@ fun Application.module() {
                 get("state") {
                     call.respond(devicesController.getDeviceState(param(call, "ref")))
                 }
+                post("set_env") {
```

Comment:
Let's make it `post("environment")` so it's more in line with other endpoints. POST verb already implies `setting`

Reply:
Done.
#34